### ISSUE TITLE
In unscored-location-dates, use two-letter abbreviations for states to match variant nowcast hub convention

### ISSUE THREAD

**Issue by elray1** [reichlab/variant-nowcast-hub#113]
Currently, the generated count files use full state names. Let's update to match the variant nowcast hub convention of two-letter state abbreviations. It may make sense to implement this change in cladetime rather than in the downstream script that generates these files?

**Comment by bsweger** [comment]
Yes, let’s make the change upstream! Will move this issue to the cladetime repo.
